charts/galoy/values.yaml

```
@@ -558,6 +558,9 @@ tracing:
   ## Jaeger Tracing Host
   ##
   jaegerHost: localhost
+  ## Prefix for tracing
+  ##
+  prefix: galoy-staging
```

I think the default `name-prefix` should be `galoy-dev`
```suggestion
  prefix: galoy-dev
```